programs/drift/src/state/user.rs

```diff
@@ -546,6 +549,118 @@ impl User {
     }
 }
 
+pub const RFQ_PDA_SEED: &str = "RFQ";
+#[zero_copy(unsafe)]
+#[derive(Default, Eq, PartialEq, Debug)]
+#[repr(C)]
+pub struct RFQOrderId {
+    pub uuid: [u8; 8],
+    pub max_ts: i64,
+}
+
+impl RFQOrderId {
+    pub fn new(uuid: [u8; 8], max_ts: i64) -> Self {
+        Self { uuid, max_ts }
+    }
+}
+
+impl Size for RFQUserAccount {
+    const SIZE: usize = 776;
+}
+
+#[account(zero_copy(unsafe))]
+#[derive(Default, Eq, PartialEq, Debug)]
+#[repr(C)]
+pub struct RFQUserAccount {
```

Comment:
1) maybe we should make this at the authority level so maker only has to do it once
2) i think you want to store the authority or user on the account so its easier to know which user the account belongs to